src/scss/05_objects/main.scss

```diff
@@ -28,19 +22,10 @@
   align-items: flex-start;
 }
 
-.o-section {
-  display: flex;
-  flex-direction: column;
-
-  h2 {
-    text-align: center;
-    line-height: 0;
-  }
-
-  hr {
-    height: 1px;
-    border: 0;
-    background-color: #BBB;
-    width: 10%;
-  }
+.o-underline {
+  border-bottom: 1px rgb(221, 221, 221) dotted;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 1.5em;
```

Comment:
You can condense these margins using the shorthand:
`margin: top right bottom left`

So assuming that margin-top is 0, you'd write:
`margin: 0 auto 1.5em auto`

You can make it even shorter with shorthand of three properities, which is:
`margin: top right-and-left bottom`
So it'd be:
`margin: 0 auto 1.5em`

Reply:
Done, thanks!